lib/Edited/prototype/listen.js

```diff
@@ -1,18 +1,42 @@
 'use strict'
-var on
-var TEXT_ALTERING_EVENT_TYPES
-
-on = require('add-event-handler')
-TEXT_ALTERING_EVENT_TYPES = require('../../text-altering-event-types')
+var on = require('add-event-handler')
+var Combokeys = require('combokeys')
 
 module.exports = function () {
   var self = this
 
-  self.listener = function (e) {
-    self.discern(e)
+  // a combokeys instance for character addition
+  self.anyCharCombokeys = new Combokeys(self.element)
+  // `any-character` is a combokeys special argument that
+  // calls back on any character
+  self.anyCharCombokeys.bind('any-character', function () {
+    self.discern('character-addition')
+  })
+  self.anyCharCombokeys.stopCallback = function (e) {
+    if (e.charCode === 32) {
+      // space is handled as a different edit type
+      return true
+    }
+    return false
   }
 
-  TEXT_ALTERING_EVENT_TYPES.forEach(function (type) {
-    on(self.element, type, self.listener)
+  // a combokeys instance for each of the "special" keyboard edit types
+  self.nonCharCombokeys = new Combokeys(self.element)
+  ;['backspace', 'del', 'space', 'enter'].forEach(function (key) {
```

Comment:
Does combokeys normalize `del` and `delete` to the same thing? `FORWARD_REMOVAL_KEYS` used both.